traveler-swift-core/TravelerKitTests/PassengerKitTests.swift

```diff
@@ -22,12 +29,34 @@ class TravelerKitTests: XCTestCase {
     }
     
     func testFlightSearch() {
-        let travelerKit = Traveler(apiKey: "XJ7B8mFnPj6O8MT4KuwzF9sg4OtxaR6w7EeytIIT", device: UIDevice.current as! Device)
+        let travelerKit = Traveler(apiKey: "TokenyToken", device: UIDevice.current as! Device, sandboxMode: true)
 
-        let query = FlightQuery(number: "SA1", date: DateFormatter.yearMonthDay.date(from: "2018/02/27")!)
+        let query = FlightQuery(number: "AC1981", date: DateFormatter.yearMonthDay.date(from: "2019/12/27")!)
         let exp = expectation(description: "Should get a response")
 
         travelerKit.flightSearch(query: query) { (flights, error) in
+            guard let flight = flights?[0] else {
+                XCTFail()
+                return
+            }
+
+            XCTAssertEqual(flight.arrivalAirport.code, "YYZ")
+            exp.fulfill()
+        }
+
+        wait(for: [exp], timeout: 2)
+    }
+
+    func testCatalog() {
+
+        let travelerkit = Traveler(apiKey: "TokenyToken", device: UIDevice.current as! Device, sandboxMode: true)
```

Comment:
This might be able to move to setup logic, its unknown at this point what traveler initialization actually does, and in the future it might actually make separate calls asynchronously

Reply:
why force cast UIDevice?

Reply:
Updated to use `MockDevice` information instead.